web/rootfs/etc/cont-init.d/10-config

```diff
@@ -85,6 +85,12 @@ if [[ ! -f /config/config.js ]]; then
         /config/config.js
     fi
 
+    if [[ $ENABLE_WEBSOCKET -eq 1 ]]; then
+        sed -i \
+          -e "s#// websocket:.*#websocket: document.location.protocol.replace('http', 'ws') + '//' + document.location.host + '/xmpp-websocket',#" \
```

Comment:
This won't work on mobile I think. Also, we have plans to move away from having an executable config file. So, how about using PUBLIC_URL for this?

Reply:
Added if PUBLIC_URL is set.